README.adoc

```diff
@@ -288,10 +288,11 @@ After the server runs, you can find your new hypermedia-driven `inventory` servi
 At the following URLs, access the `inventory` service that is now driven by hypermedia:
 
 * http://localhost:9080/inventory/hosts[http://localhost:9080/inventory/hosts^]
-* http://localhost:9080/inventory/hosts/localhost[http://localhost:9080/inventory/hosts/{hostname}^]
+* http://localhost:9080/inventory/hosts/localhost[http://localhost:9080/inventory/hosts/localhost}^]
```

Comment:
- [ ] `}` at the end of URL should not be here
    ![image](https://user-images.githubusercontent.com/46586499/73955993-43708100-48d2-11ea-81a6-09a6d53871f6.png)